apps/aragon-fundraising/app/src/components/NewOrder/Order.js

```diff
@@ -104,9 +104,17 @@ const Order = ({ isBuyOrder }) => {
     return collateralItems[selectedCollateral].reserveRatio
   }
 
+  const getUserBalance = () => {
+    const balance = isBuyOrder ? [userDaiBalance, userAntBalance][selectedCollateral] : userBondedBalance
+    return formatBigNumber(balance, bondedDecimals, 4)
```

Comment:
The decimals of each tokens can be different.
So you should do the same you do with the balance to select the right decimal.
Something like:
```
const decimals = isBuyOrder ? collateralItems[selectedCollateral].decimals : bondedDecimals
```